content/presence-occupancy/presence.textile

```diff
@@ -0,0 +1,729 @@
+
+A @response@ object will have @response.success@ set to true if the batch presence request was successful, and false if at least one individual presence request failed. The @response.statusCode@ will be set to 200 for success, 400 for partial success and 401 for an expected failure. @response.errorCode@ will then contain the Ably error code and @response.errorMessage@ will contain the details of the error.
+
+@response.items@ will contain a list of responses for each channel for which presence has been requested in a successful response. @response.items.batchResponse@ will contain a list of each channel's results, be it an error or a success for a partially successful response. The @response.errorCode@ will always be @40020@ for a partial success.
+
+The following is an example of handling the various responses:
+
+```[javascript]
+var ablyRest = new Ably.Rest({ key: '{{API_KEY}}' })
+var content = { "channel": "channel1,channel2" }
+ablyRest.request('GET', '/presence', content, null, {}, function(err, response) {
+  if(err) {
+    // Throw error
+  } else {
+    if(response.success) {
+      // If complete success
+      for(i = 0; i < response.items.length; i++) {
+        // Each response item will be roughly of the style:
+        /*
+          {
+            "channel": "channel1",
+            "presence": [
+              { "action": 1, "clientId": "CLIENT1" },
+              { "action": 1, "clientId": "CLIENT2" }
+            ]
+          }
+        */
+      }
+    } else if(response.errorCode === 40020) {
+      // If partial success
+      for(i = 0; i < response.items[0].batchResponse.length; i++) {
+        // Each batchResponse item will either be the same as success if it succeeded, or:
+        /*
+          {
+            "channel": "channel1",
+            "error": {
+              "code": 40160,
+              "message": "ERROR_MESSAGE",
+              "statusCode": 401
+            }
+          }
+        */
+      }
+    } else {
+      // If failed, check why
+      console.log(response.errorCode + ', ' + response.errorMessage);
+    }
+  }
+});
+```
+
+h2(#presence-history). Presence history
+
+"History":/storage-history/history provides access to instantaneous "live" history as well as the longer term persisted history for presence channels. If "persisted history":/storage-history/history#presence-history is enabled for the channel, then presence events will typically be stored for 24 or 72 hours. If persisted history is not enabled, Ably retains the last two minutes of presence event history in memory.
+
+Every message that is persisted to or retrieved from disk counts as an extra message towards your monthly quote. For example, in a persistence enabled channel a published message will deduct two messages from your monthly quota. If the message is later retrieved from history, another message will be deducted from your monthly quota.
```

Comment:
Is this paragraph covered in the history section? If so, I don't think we need to include it here too.